src/cdk-experimental/popover-edit/table-directives.ts

```diff
@@ -168,7 +170,8 @@ export class CdkPopoverEdit<C> implements AfterViewInit, OnDestroy {
 
   constructor(
       protected readonly services: EditServices, protected readonly elementRef: ElementRef,
-      protected readonly viewContainerRef: ViewContainerRef) {}
+      protected readonly viewContainerRef: ViewContainerRef,
+      @Optional() private _dir?: Directionality) {}
```

Comment:
Can this go in EditServices instead? See #15797

Reply:
I'm not sure we want that, because the direction should come from the trigger's scope. Different sections of the app can have different layout directions.

Reply:
It depends how granular this needs to be - EditServices is provided by the table element - so unless we think people will be swapping text directions at the individual cell level it should be fine.

Reply:
I'm inclined to try it with EditServices and see if it ever comes up as an issue in the real world.

Reply:
Changed.